### PR TITLE
Add support for back button accessibility trait. 

### PR DESCRIPTION
**Pull request by RoyalPineapple**
Navigation controller back buttons contain a secret menu trait which calls them out with a distinct utterance of "back button" rather than simply "button". This change adds support for detecting this trait and provides the matching utterance string.

### CODE REVIEW

**Review by kyleve — APPROVED** [review]
Nice, thanks!

File: Sources/AccessibilitySnapshot/Core/Swift/Assets/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -10,6 +10,9 @@
 /* Description for the 'button' accessibility trait */
 "trait.button.description" = "Taste.";
 
+/* Description for the 'back button' accessibility trait */
+"trait.backbutton.description" = "Zurück Taste.";
```
Comment (kyleve) [inline Sources/AccessibilitySnapshot/Core/Swift/Assets/de.lproj/Localizable.strings:14]: image

**Review by NickEntin — COMMENTED** [review]
Logic looks good! Can we add a snapshot test or two (maybe of a nav controller with a second vc pushed, with and without a title on the previous item) to test this?

File: Example/SnapshotTests/ReferenceImages/_64/SnapshotTests.AccessibilitySnapshotTests/testNavBarBackButtonTraitsWithTitles_16_4_393x852@3x.png
Comment (kyleve) [inline Example/SnapshotTests/ReferenceImages/_64/SnapshotTests.AccessibilitySnapshotTests/testNavBarBackButtonTraitsWithTitles_16_4_393x852@3x.png:1]: Yay, thanks for doing this!

**Comment by RoyalPineapple** [comment]
> Logic looks good! Can we add a snapshot test or two (maybe of a nav controller with a second vc pushed, with and without a title on the previous item) to test this?

@NickEntin I added snapshot tests as you suggested. Do you mind a re-review?

File: Example/AccessibilitySnapshot/NavBarBackButtonAccessibilityTraitsViewController.swift [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+        return view as! View
+    }
+
+    // MARK: - UIViewController
+
+    override func loadView() {
+        view = View(titles: titles)
+    }
+
+    override func viewDidLoad() {
```
Comment (kyleve) [inline Example/AccessibilitySnapshot/NavBarBackButtonAccessibilityTraitsViewController.swift:47]: Nit: Delete?

File: Example/AccessibilitySnapshot/NavBarBackButtonAccessibilityTraitsViewController.swift [hunk 3]
```diff
@@ -0,0 +1,371 @@
...
+            middleLabel.capInsetsAlignmentProxy.align(withSuperview: .center)
+        }
+
+    }
+
+}
+
+// MARK: -
+
+private final class TabBarTraitContainerView: UIView {
```
Comment (kyleve) [inline Example/AccessibilitySnapshot/NavBarBackButtonAccessibilityTraitsViewController.swift:286]: Do we need this?

File: Example/AccessibilitySnapshot/NavBarBackButtonAccessibilityTraitsViewController.swift [hunk 4]
```diff
@@ -0,0 +1,371 @@
...
+        }
+        
+
+    }
+
+}
+
+// MARK: -
+
+private final class TabBarTraitView: UIView {
```
Comment (kyleve) [inline Example/AccessibilitySnapshot/NavBarBackButtonAccessibilityTraitsViewController.swift:136]: Do we need this?

**Review by kyleve — APPROVED** [review]
LGTM!

**Review by meherkasam — APPROVED** [review]

**Pull request merged**